traffic_server/tsb/trafficserver.spec

```diff
@@ -0,0 +1,151 @@
+cp -r /opt/trafficserver/openssl/lib $RPM_BUILD_ROOT/opt/trafficserver/openssl/lib
+
+mkdir -p $RPM_BUILD_ROOT/opt/trafficserver/lua
+cp -r /opt/build/luacrypto/src/.libs/crypto.so $RPM_BUILD_ROOT/opt/trafficserver/lua
+
+%clean
+rm -rf $RPM_BUILD_ROOT
+
+%pre
+id ats &>/dev/null || /usr/sbin/useradd -u 176 -r ats -s /sbin/nologin -d /
+
+%post
+/bin/systemctl daemon-reload
+/bin/systemctl enable trafficserver
+
+%preun
+/bin/systemctl stop trafficserver
+
+# if 0 uninstall, if 1 upgrade
+if [ "$1" = "0" ]; then
+	/bin/systemctl disable trafficserver
+fi
+
+%postun
+# Helpful in understanding order of operations in relation to install/uninstall/upgrade:
+#     https://fedoraproject.org/wiki/Packaging:Scriptlets
+
+# Always do this because the service file may have been updated.
+/bin/systemctl daemon-reload
+
+# if 0 uninstall, if 1 upgrade
+if [ "$1" = "0" ]; then
+	id ats &>/dev/null && /usr/sbin/userdel ats
+fi
+
+%files
+%defattr(-,root,root)
+%attr(644,-,-) /usr/lib/systemd/system/trafficserver.service
+%dir /opt/trafficserver
+/opt/trafficserver/openssl
+/opt/trafficserver/lua
+/opt/trafficserver/bin
+/opt/trafficserver/include
+/opt/trafficserver/lib
+/opt/trafficserver/libexec
+/opt/trafficserver/share
+%dir /opt/trafficserver/var
+%attr(-,ats,ats) /opt/trafficserver/var/trafficserver
+%dir /opt/trafficserver/var/log
+%attr(-,ats,ats) /opt/trafficserver/var/log/trafficserver
+%dir /opt/trafficserver/etc
+%attr(-,ats,ats) %dir /opt/trafficserver/etc/trafficserver
+%attr(-,ats,ats) %dir /opt/trafficserver/etc/trafficserver/snapshots
+/opt/trafficserver/etc/trafficserver/body_factory
+/opt/trafficserver/etc/trafficserver/trafficserver-release
+%config(noreplace) %attr(644,ats,ats) /opt/trafficserver/etc/trafficserver/cache.config
+%config(noreplace) %attr(644,ats,ats) /opt/trafficserver/etc/trafficserver/hosting.config
+%config(noreplace) %attr(644,ats,ats) /opt/trafficserver/etc/trafficserver/ip_allow.config
+%config(noreplace) %attr(644,ats,ats) /opt/trafficserver/etc/trafficserver/logging.yaml
+%config(noreplace) %attr(644,ats,ats) /opt/trafficserver/etc/trafficserver/parent.config
+%config(noreplace) %attr(644,ats,ats) /opt/trafficserver/etc/trafficserver/plugin.config
+%config(noreplace) %attr(644,ats,ats) /opt/trafficserver/etc/trafficserver/records.config
+%config(noreplace) %attr(644,ats,ats) /opt/trafficserver/etc/trafficserver/remap.config
+%config(noreplace) %attr(644,ats,ats) /opt/trafficserver/etc/trafficserver/socks.config
+%config(noreplace) %attr(644,ats,ats) /opt/trafficserver/etc/trafficserver/splitdns.config
+%config(noreplace) %attr(644,ats,ats) /opt/trafficserver/etc/trafficserver/ssl_multicert.config
+%config(noreplace) %attr(644,ats,ats) /opt/trafficserver/etc/trafficserver/ssl_server_name.yaml
+%config(noreplace) %attr(644,ats,ats) /opt/trafficserver/etc/trafficserver/storage.config
+%config(noreplace) %attr(644,ats,ats) /opt/trafficserver/etc/trafficserver/volume.config
+
+%changelog
```

Comment:
Do we want this full changelog in here, its  fairly out of date anyway plus I might consider this going out to opensource to be a fresh start

Reply:
I considered it, but I'm always leery of removing attribution notices. These changelog entries document the work of the people who's work I'm submitting. I _could_ remove them, but I think I'd prefer to keep them.